www/src/components/sidebar/scroll-sync-sidebar.js

```diff
@@ -64,27 +64,18 @@ class ScrollSyncSection extends Component {
   }
 }
 
-// @todo make recursive
-const _getItemIds = itemList => {
+const _getItemIds = section => {
   let list = []
 
-  itemList.forEach(section => {
-    if (section.items) {
-      let sectionItems = section.items
-        .map(item => {
-          let subItemIds = []
-          if (item.items) {
-            subItemIds = item.items.map(subitem => subitem.hash)
-          }
-          return [item.hash, ...subItemIds]
-        })
-        .reduce((prev, current) => prev.concat(current))
+  section.forEach(section => {
+    if (section.hasOwnProperty('hash')) list.push(section.hash)
 
-      list.push(sectionItems)
+    if (section.items) {
+      return list.push(..._getItemIds(section.items))
```

Comment:
@m-allanson I agree.
Actually, this doesn't even need to return anything for this to work.